assisted-installer/software-raid/README.md

```diff
@@ -0,0 +1,248 @@
+# **Deploying an OpenShift Cluster with software raid using Assisted Installer**
+
+This document briefly describes how to run a deployment of an OpenShift Cluster through Assisted Installer where software raid needs to be set up.
+
+We will use [Assisted Installer CLI](https://github.com/karmab/aicli) in order to interact with the Assisted Service API in a CLI fashion.
+
+The approach involves using the API to inject an extra ignition config on the hosts that will trigger the configuration of the raid.
+
+## **Preparing the ignition file content**
+
+We prepare a file to indicate which software raid is to be configured, and which disks are part of it.
+
```

Comment:
Maybe add a note here explaining that this example uses vda and vdb, and that it may be different in other envs.